docs/code-samples/getting-started/daml/Scripts/Holding.daml

```diff
@@ -130,44 +111,28 @@ runHolding = do
     alice
     bank
     bob
-    public
     aliceAccount
     bobAccount
     cashInstrument = instrumentKey
-    holdingFactoryCid
+    tokenFactoryCid
     aliceCashHoldingCid
 
+-- | Helper container used to transfer state to the next script.
+data HoldingState = HoldingState
```

Comment:
Curious as to why you moved the definition of `HoldingState` to here? It means it's defined after its usage in both the script signature on line 27 and its actual usage on line 110.

Reply:
I've noticed you've also moved the other `State` definitions to the bottom of each file.

Reply:
`HoldingState` has no significance in the context of the `Holding` tutorial, as it is used to carry state to the next script (`Transfer` tutorial). 

I didn't want to burden the reader with that information, but rather go directly to the script, which is the part relevant to the `Holding` tutorial.

Reply:
Makes sense 👍